README.md

```diff
@@ -116,14 +116,14 @@ Testing is only supported on Linux with bash.
 
 ## Post-Quantum Algorithms
 
-Starting with wolfSSL version 5.1.0 and nginx version 1.21.4, You can now enable the integration of liboqs in wolfSSL thus enabling post-quantum algorithms for your HTTPS connections over TLS 1.3.
+You can now enable the use of post-quantum algorithms for your HTTPS connections over TLS 1.3.
```

Comment:
I like the previous listing of wolfSSL and nginx version that the PQC support was introduced. Could we update it for the wolfSSL version that contains native implementations?

Reply:
Yes!  I can add the versions that I tested against.  Hold on.